internal/impl/peerdb/input_peerdb_integration_test.go

```diff
@@ -0,0 +1,155 @@
+package peerdb
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/ory/dockertest/v3"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	_ "github.com/lib/pq"
+
+	"github.com/benthosdev/benthos/v4/public/service"
+	"github.com/benthosdev/benthos/v4/public/service/integration"
+)
+
+func TestPeerDBIntegration(t *testing.T) {
+	// Skip test if integration tests are disabled
+	integration.CheckSkip(t)
+	t.Parallel()
+
+	// Create a new docker pool
+	pool, err := dockertest.NewPool("")
+	require.NoError(t, err, "Could not connect to docker")
+	pool.MaxWait = 3 * time.Minute
+
+	// Run a postgres container
+	resource, err := pool.RunWithOptions(&dockertest.RunOptions{
+		Repository: "debezium/postgres",
```

Comment:
Is this significant? Would the stock postgres image be missing extensions..? 🤔

Reply:
This image has some changes to the default postgres image, it isn't hard to get this in there but was more convenient to just is it, specifically

```
wal_level = logical
max_wal_senders = <N > 0>
max_replication_slots = < N > 0>
```